internal/services/applicationinsights/validate/workbook_template_id.go

```diff
@@ -0,0 +1,21 @@
+package validate
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/applicationinsights/2020-11-20/applicationinsights"
+)
+
+func WorkbookTemplateID(input interface{}, key string) (warnings []string, errors []error) {
```

Comment:
this is available within the sdk (`applicationinsights.ValidateWorkbookTemplateID`) - so we can remove this?